src/cpp/src/utils.cpp

```diff
@@ -183,6 +174,22 @@ ov::genai::OptionalGenerationConfig get_config_from_map(const ov::AnyMap& config
         return std::nullopt;
 }
 
+ProcessorConfig from_any_map(
+    const ov::AnyMap& config_map,
+    const ProcessorConfig& initial
+) {
```

Comment:
I would keep this method somewhere closer to VLM pipeline as this function does not represent "common utils"

Reply:
resolved in https://github.com/Wovchena/openvino.genai-public/pull/68